pattern-2/apim-gw/wso2apim-gateway-deployment.yaml

```diff
@@ -31,7 +31,7 @@ spec:
     spec:
       containers:
       - name: wso2apim-gateway
-        image: docker.wso2.com/wso2am:2.5.0
+        image: docker.wso2.com/wso2am:2.6.0
         livenessProbe:
```

Comment:
IMO, we should define the resource requirements and limits for API Manager based on the recommendations [1].

WDYT?

[1]: [Resource requirements and limits](https://github.com/wso2/kubernetes-apim/blob/2.2.x/pattern-1/apim/wso2apim-deployment.yaml#L56)